.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,23 @@
+name: Docker Image CI
+
+on: [push]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    # Check on tutorial docker containers 
+    - run: docker pull quay.io/dockstore-testing/dockstore-tool-md5sum
+    - run: docker pull rancher/cowsay
+    - run: docker pull quay.io/ldcabansay/samtools
+    # this container is built by the user, not pulled from docker hub
+    # - run: docker pull tabix:latest
```

Comment:
Do we need to keep these commented out lines? If yes, some comments about why would be helpful

Reply:
Kinda. I found these in a text search and they look like containers. Will elaborate